docs/tutorials/00-landscaper-installation.md

```diff
@@ -0,0 +1,124 @@
+# Install the Landscaper
+
+This tutorial describes how to install the Landscaper on a Kubernetes cluster and optionally how to install an OCI registry
+on a Gardener Shoot cluster.
+
+### Prerequisites
+
+- Kubernetes cluster
+- Landscaper CLI (see the [installation documentation](https://github.com/gardener/landscapercli/blob/master/docs/installation.md))
+- This tutorial assumes that the landscaper will be installed in the namespace `landscaper`. The namespace has to created before the landscaper is being installed.
```

Comment:
in our examples and tests we use the namespace `ls-system`. Where is this namespace defined? in the cli?
We should stick to one to not confuse people.